task0.sql

```diff
@@ -0,0 +1,132 @@
+CREATE DATABASE BanksDB
+CREATE TABLE social_statuses(
+	social_status_id INT IDENTITY(1,1) PRIMARY KEY NOT NULL,
+	type VARCHAR(25)
+);
+
+CREATE TABLE banks(
+	bank_id INT IDENTITY(1,1)  PRIMARY KEY NOT NULL,
+	bank_name VARCHAR(25)
+);
+
+CREATE TABLE accounts(
+	account_id INT IDENTITY(1,1)  PRIMARY KEY NOT NULL,
+	account_number VARCHAR(25),
+	balance REAL
+);
+
+CREATE TABLE clients(
+	client_id INT IDENTITY(1,1) PRIMARY KEY NOT NULL,
+	bank_id INT  NOT NULL,
+	account_id INT NOT NULL UNIQUE,
+	social_status_id INT  NOT NULL,
+	first_name  VARCHAR(25),
+	last_name  VARCHAR(25),
+	patronymic  VARCHAR(25),
+	birth_date DATE,
+
+	CONSTRAINT FK_Clients_To_Accounts FOREIGN KEY (account_id) REFERENCES accounts (account_id),
+	CONSTRAINT FK_Clients_To_Banks FOREIGN KEY (bank_id) REFERENCES banks (bank_id),
+	CONSTRAINT FK_Clients_To_Social_Statuses FOREIGN KEY (social_status_id) REFERENCES social_statuses (social_status_id)
+);
+
+CREATE TABLE cards(
+	card_id INT IDENTITY(1,1)  PRIMARY KEY NOT NULL,
+	account_id INT NOT NULL,
+	card_number INT,
+	validity_period DATE,
+    balance REAL
+	CONSTRAINT FK_Cards_To_Accounts FOREIGN KEY (account_id) REFERENCES accounts (account_id)
+);
+
+CREATE TABLE cities(
+	city_id INT IDENTITY(1,1)  PRIMARY KEY NOT NULL,
+	city_name VARCHAR(25)
+);
+
+CREATE TABLE branches(
+	branch_id INT IDENTITY(1,1)   PRIMARY KEY NOT NULL,
+	bank_id INT NOT NULL,
+	phone_number VARCHAR(25),
+	address VARCHAR(50),
+	CONSTRAINT FK_Branches_To_Banks FOREIGN KEY (bank_id) REFERENCES banks (bank_id)
+);
+
+CREATE TABLE branch_city(
+	branch_city_id INT IDENTITY(1,1) PRIMARY KEY ,
+	branch_id INT NOT NULL,
+	city_id INT NOT NULL,
+	CONSTRAINT FK_Branches_Cities_To_Cities FOREIGN KEY (city_id) REFERENCES cities (city_id),
+	CONSTRAINT FK_Branches_Cities_To_Branches FOREIGN KEY (branch_id) REFERENCES branches (branch_id)
```

Comment:
You can have several branches in one city
But you couldn't have one branch in different cities
This table created for many-to-many connection
But here should be one-to-many
Remove this table